types/should-sinon/index.d.ts

```diff
@@ -89,3 +85,12 @@ declare module 'sinon' {
         threw(ex: string | Error): void;
     }
 }
+
+// keep backwards compat with earlier DefinitelyTyped release which made these
+// types available
+declare module 'sinon' {
+    interface SinonSpyCallApi {
```

Comment:
This could be removed, but leaving it maintains backwards compat in case someone did `import { type ShouldSinonAssertion } from 'sinon';`. Not sure how strict DefinitelyTyped is on that?